.github/workflows/update-platform-version.yml

```diff
@@ -58,6 +58,7 @@ jobs:
     with:
       base_tag: master
       tags: master,develop,${{ needs.call-bump-version.outputs.new_version }}
+      branch: ${{ needs.call-merge-release.inputs.base }}
```

Comment:
Not sure this will work. I think we can only use outputs not inputs? Here it should be fine to hard code `master` as the ref